src/renderer/screens/nft/Gallery/TokensList/Item.js

```diff
@@ -47,9 +48,10 @@ type Props = {
   tokenId: string,
   id: string,
   mode: "grid" | "list",
+  withContextMenu?: boolean,
 };
 
-const Row = ({ contract, tokenId, id, mode, account }: Props) => {
+const Row = ({ contract, tokenId, id, mode, account, withContextMenu = false }: Props) => {
```

Comment:
What do you think of using the isGrid value instead of using a new prop?
Since they have the same value (mode === "grid").

This way we isolate this knowledge about the context menu in this component and the parent doesn't need to know about it. (in grid mode = context menu with right click in the component, not in grid mode = context menu with left click in the three dots)

Reply:
I actually don't think we should link directly features and design, even though I agree with you that _for the moment_ it's our only use case for this functionality. 

The context menu is actually the ability to right click on the NFT itself not on the triple dots. So we could cleary at some point decide that we want to display an NFT somewhere in grid mode with no context menu on it, and vice-versa, display multiple nfts in list mode that would also be right-clickable to display the links and not only through a left-click on the triple dots.

wdyt ?

Reply:
Hmmm yeah, I see your point. Ok, let's go for it 👍